### PR TITLE
add versioning

### PR DESCRIPTION
**Pull request by vesperusiak**

### CODE REVIEW

File: getting-started/versioning.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+# Versioning
+
+### Format
+
+`v<major>.<minor>`
+
+examples: `v0.1` `v0.2`
+
+All components of this API have single, common version.
```
Comment (retrovampire) [inline getting-started/versioning.md]: All components of this API have single, common version. -> All components of this API have a single common version.

File: getting-started/versioning.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+title: "Versioning"
+weight: 100
+---
+
+# Versioning
+
```
Comment (retrovampire) [inline getting-started/versioning.md:7]: tu nie będzie żadnego intro?

File: getting-started/versioning.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+`v<major>.<minor>`
+
+examples: `v0.1` `v0.2`
+
+All components of this API have single, common version.
+
+### Endpoints
+
+Our components have different endpoints for every version. For example here is what url with version looks like for customer chat API:
```
Comment (retrovampire) [inline getting-started/versioning.md]: Our components have different endpoints for every version. For example here is what url with version looks like for customer chat API: -> Our components have different endpoints for every version. For example here is what a URL with a version looks like for the customer chat API:

File: getting-started/versioning.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+`api.chat.io/customer/v0.2/<req>`
+
+or if you prefer to point always to recent version (not recomended):
```
Comment (retrovampire) [inline getting-started/versioning.md]: (not recomended): -> (not recommended):

File: getting-started/versioning.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+`api.chat.io/customer/<req>`
+
+### Change policy
+
+Before version `1.0` new versions may introduce breaking changes. We don't know already if that policy will hold after `1.0`. We are holding old versions for unknown amount of time. We will add an information that a version is deprecated in this documentation before version removal.
```
Comment (retrovampire) [inline getting-started/versioning.md]: Before version `1.0` new versions may introduce breaking changes. We don't know already if that policy will hold after `1.0`. We are holding old versions for unknown amount of time. We will add an information that a version is deprecated in this documentation before version removal. -> Before we reach (albo: before the API reaches) the `1.0` version, new versions may introduce breaking changes. We don't know yet if that policy holds after `1.0`. We support old versions for an undefined period of time. We will add a notice in this documentation that a version is deprecated before it's removed.

File: getting-started/versioning.md (hunk 2 above)
Comment (vesperusiak) [reply getting-started/versioning.md]: Wydaje mi się, że jest zbędne w tej konkretnej sekcji

File: getting-started/versioning.md (hunk 1 above)
Comment (vesperusiak) [reply getting-started/versioning.md]: pojawiła się wątpliwość, ze jak to zdanie znajdzie się w sekcji Format to może zostać źle zrozumiane, stąd przenoszę to do wstępu

**Pull request closed without merging**